Description: Change symbolic links
 Change the way symbolic links are created.
 .
 a53 (1.0-1) precise; urgency=low
 .
   * Initial release. (Closes: #XXXXXX)
Author: Anuvind Menon <xray@xrayBase>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- a53-1.0.orig/Makefile
+++ a53-1.0/Makefile
@@ -12,8 +12,8 @@ liba53.so.1.0: ${SOURCE_FILES} ${INCLUDE
 # install A5/3 library
 install: liba53.so.1.0
 	cp liba53.so.1.0 $(DESTDIR)/usr/lib
-	ln -sf $(DESTDIR)/usr/lib/liba53.so.1.0 $(DESTDIR)/usr/lib/liba53.so.1
-	ln -sf $(DESTDIR)/usr/lib/liba53.so.1.0 $(DESTDIR)/usr/lib/liba53.so
+	ln -sf $(DESTDIR)/usr/lib/liba53.so.1.0 /usr/lib/liba53.so.1
+	ln -sf $(DESTDIR)/usr/lib/liba53.so.1.0 /usr/lib/liba53.so
 	cp a53.h $(DESTDIR)/usr/include
 
 # test installed A5/3 library
